crates/core/plugin_sm/src/plugin.rs

```diff
@@ -417,24 +426,12 @@ impl Plugin for ExternalPluginCommand {
         let command = self.command(LIST, None)?;
         let output = self.execute(command, logger).await?;
         if output.status.success() {
-            let mut software_list = Vec::new();
             let mut rdr = ReaderBuilder::new()
                 .has_headers(false)
                 .delimiter(b'\t')
+                .flexible(true)
                 .from_reader(output.stdout.as_slice());
```

Comment:
This piece of code should be included in the `deserialize_module_info` function. This is a key part of the contract between a plugin and the agent.